projects/schematics/src/migrations/5_0/constructor-deprecations/data/cart-lib-checkout.migration.ts

```diff
@@ -0,0 +1,2679 @@
+      },
+    ],
+    addParams: [],
+  },
+  {
+    class: 'OccReplenishmentOrderNormalizer',
+    importPath: '@spartacus/core',
+    deprecatedParams: [
+      {
+        className: 'ConverterService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'OrderEntryPromotionsService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'OrderEntryPromotionsService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [],
+  },
+  {
+    class: 'OrderReturnRequestService',
+    importPath: '@spartacus/core',
+    deprecatedParams: [
+      {
+        className: 'Store',
+        importPath: '@ngrx/store',
+      },
+      {
+        className: 'UserIdService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'Store',
+        importPath: '@ngrx/store',
+      },
+    ],
+  },
+  {
+    class: 'SelectiveCartService',
+    importPath: '@spartacus/core',
+    deprecatedParams: [
+      {
+        className: 'Store',
+        importPath: '@ngrx/store',
+      },
+      {
+        className: 'UserService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'MultiCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'BaseSiteService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'CartConfigService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'UserIdService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'Store',
+        importPath: '@ngrx/store',
+      },
+      {
+        className: 'MultiCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'CartConfigService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'MultiCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'WishListService',
+    importPath: '@spartacus/core',
+    deprecatedParams: [
+      {
+        className: 'Store',
+        importPath: '@ngrx/store',
+      },
+      {
+        className: 'UserService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'MultiCartService',
+        importPath: '@spartacus/core',
+      },
+      {
+        className: 'UserIdService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    removeParams: [
+      {
+        className: 'MultiCartService',
+        importPath: '@spartacus/core',
+      },
+    ],
+    addParams: [
+      {
+        className: 'MultiCartFacade',
+        importPath: '@spartacus/cart/base/root',
+      },
+    ],
+  },
+  {
+    class: 'ConsignmentTrackingComponent',
+    importPath: '@spartacus/order/components',
+    deprecatedParams: [
+      {
+        className: 'OrderFacade',
```

Comment:
awesome

Reply:
same for other changes here related to order-lib. There's migrations for 4.0,4.1 below here and migrations for when it lived in order lib starting from 4.2